### PR TITLE
dev-db/sadisplay: new package

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.5, Repoman-2.3.2

### CODE REVIEW

**Comment by TheChymera** [comment]
Does [this](https://bpaste.net/show/d1f2b12afbf1) look like the package isn't working on python 3? I have actually used it on python 3 already, and I have not encountered any issues.

**Comment by mgorny** [comment]
Well, it looks more like python2.7 setuptools modules got somehow loaded in python3. I'll try to reproduce it to see whether it's a problem in the package or your system is broken ;-).

File: dev-db/sadisplay/sadisplay-0.4.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+RDEPEND="dev-python/sqlalchemy[${PYTHON_USEDEP}]"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? ( ${RDEPEND} )
+	"
+S="${WORKDIR}/estin-${PN}-${COMMIT_ID}"
+
+python_test() {
+	distutils_install_for_testing
```
Comment (mgorny) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: Is it really needed? Tests pass fine without it for me.

File: dev-db/sadisplay/sadisplay-0.4.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	nosetests -v || die
+}
+
```
Comment (mgorny) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: Stray whitespace.

File: dev-db/sadisplay/sadisplay-0.4.8.ebuild (hunk 1 above)
Comment (TheChymera) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: For me ti changes the error message for the python 3 test fail :-/

Comment (mgorny) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: As I said, you probably have a broken system. If you used pip, you're doomed and need to clean up the mess it installed. If you haven't, looked for rogue `*.pth` files in `/usr/lib*/python*/site-packages`.

Comment (TheChymera) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: I never use pip as superuser - likely portage messed something up, I'll try to debug that first, I also seem to be getting some matplotlib issues all of a sudden...

Comment (mgorny) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: Maybe look for setuptools-related stuff.

Comment (TheChymera) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: @mgorny  very strange, it seems it's an issue with testing it with `ebuild sadisplay-0.4.8.ebuild test` (it keeps giving me that error), but if I drag and drop it in my local overlay and then do `FEATURES="test" emerge -av sadisplay` the tests pass just fine.

The other errors were apparently unrelated (I get weird `ImportError: Gtk3 backend requires pygobject to be installed.` errors if I import `matplotlib.plt` after `pandas`).

Comment (mgorny) [inline dev-db/sadisplay/sadisplay-0.4.8.ebuild]: The `ebuild` tool is unreliable, please always test stuff with `emerge`.

**Pull request closed without merging**